signer/src/lib.rs

```diff
@@ -33,8 +33,8 @@ pub mod sr25519;
 pub mod ecdsa;
 
 // An ethereum signer implementation.
-#[cfg(feature = "eth")]
-#[cfg_attr(docsrs, doc(cfg(feature = "eth")))]
+#[cfg(feature = "unstable-eth")]
+#[cfg_attr(docsrs, doc(cfg(feature = "unstable-eth")))]
```

Comment:
I thought it'd be better to play it a little safe and call this unstable for now, so we can merge it quicker and work on the interface or whatever later if we need to.